pkg/kudoctl/cmd/package.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/bundle"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+const (
+	example = `
+		The package argument must be a directory which contains the operator definition files.  The package command will create a tgz file containing the operator.
+
+		# package zookeeper (where zookeeper is a folder in the current directory)
+		kubectl kudo package zookeeper
+
+		# Specify a destination folder other than current working directory
+		kubectl kudo package ../operators/repository/zookeeper/operator/ --destination=out-folder`
+)
+
+type packageCmd struct {
```

Comment:
this should be named options to be in line with other commands

Reply:
I much prefer the cli model that helm uses... which aligns with this style.   I plan to refactor the other commands to align with this one.

Reply:
should we maybe agree on that first? 😄

Reply:
we have a lot of work to do in the CLI when it comes to naming... for instance `defaultOptions` https://github.com/kudobuilder/kudo/blob/master/pkg/kudoctl/cmd/upgrade.go#L42
makes no sense... as this var lives in the package space and is likely the upgradeOptions or defaultUpgradeOptions.

Reply:
Oh yeah, defaultOptions for sure, that should be renamed. But I like the consistency of having those options classes for each command...